### PR TITLE
Add description of ""close_comm_on_error".

### PR DESCRIPTION
**Pull request by janiversen**
## Proposed change

Add description of new configuration attribute in modbus.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
https://github.com/home-assistant/core/pull/50583 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by janiversen** [comment]
I do not get it !  I have done:
git checkout next
git pull
git checkout issue_reset_socket
git rebase next
git push -f

AND it still have the label that I need to rebase.

**Comment by MartinHjelmare** [comment]
The label bot isn't that smart (yet).

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -48,6 +48,11 @@ modbus:
 ```
 
 {% configuration %}
+close_comm_on_error:
+  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters sends garble to HA when opened, this lead to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```
Comment (MartinHjelmare) [inline source/_integrations/modbus.markdown]:
```suggestion
  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters sends garble to HA when opened, this leads to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_integrations/modbus.markdown [hunk 2]
```diff
@@ -48,6 +48,11 @@ modbus:
 ```
 
 {% configuration %}
+close_comm_on_error:
+  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters sends garble to HA when opened, this leads to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```
Comment (frenck) [inline source/_integrations/modbus.markdown]:
```suggestion
  description: Determines if the modbus communication is closed when an error occurs, default is true. Some serial-rs485 adapters send garble to HA when opened, this leads to a disconnect and a new connect, which can continue. If in a running communication the debug log contains a message from pymodbus, with the text "cleaning....", then try this parameter.
```

**Comment by janiversen** [comment]
heh, I looked at the bot code, sometime ago, because balloob "challenged" me, but that is a language completely strange 👎

File: source/_integrations/modbus.markdown (hunk 2 above)
Comment (janiversen) [reply source/_integrations/modbus.markdown]: I will not let me accept your suggestion, but please commit the change.

Comment (frenck) [reply source/_integrations/modbus.markdown]: I already committed it, hence you couldn't accept it 😉

**Review by frenck — APPROVED** [review]
Thanks, @janiversen 👍

**Pull request merged**